src/modules/launcher/Plugins/Microsoft.PowerToys.Run.Plugin.Service/Properties/Resources.resx

```diff
@@ -139,7 +139,10 @@
     <value>Restart (Ctrl+R)</value>
   </data>
   <data name="wox_plugin_service_restarted_notification" xml:space="preserve">
```

Comment:
Why `wox_plugin` and not `PowerToysRun_Plugin` or `Microsoft_Plugin`?

_(The same question for the other strings too?)_